### ISSUE TITLE
Is there going to be another image released?

### ISSUE THREAD

**Issue by galliangg** [moritzmhmk/buildroot-camera-rpi#7]
Hi,

I do use this and I wondered if there would be a new release of the image?

Thanks

**Comment by moritzmhmk** [comment]
Yes (I do also still use this), but I cant promise any release date currently. Probably not this year.

**Comment by galliangg** [comment]
Thank you! No problem, the current version works for me, I just like to be updated. 
Thank you for making this project

**Comment by Pairan** [comment]
What will be included in a new version?

**Comment by moritzmhmk** [comment]
Probably only updated packages.

**Comment by Pairan** [comment]
currently I wont get the Cam to respond when Im outside of the LAN.  All other components react normal (as they talk via the appleTv).

is this something that can be fixed or is it a RTFM-Problem that is caused by my idea of how I would assume it to work ? :D

**Comment by moritzmhmk** [comment]
The camera requires an Apple TV 4th generation or newer, is yours older?

**Comment by Pairan** [comment]
nope ... 4th gen here ... but it wont work while im not in the local net. 

"Diese Kamera anwortet nicht" ... :)

**Comment by moritzmhmk** [comment]
It should work - no idea why it doesn't...

**Comment by galliangg** [comment]
It works for me over the internet with an AppleTV 4th gen

**Comment by galliangg** [comment]
Hi, is it possible to get an updated image for this too?

**Comment by moritzmhmk** [comment]
An update is planned - probably in july.
